.github/ISSUE_TEMPLATE/question.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: Question
+description: Ask a question.
+title: "[QUESTION] Title"
+labels: Question
+body:
+- type: markdown
+  attributes:
+    value: |
+      If the question is related to AMD hardware, please mention [Nathan Fontenot](https://github.com/nfont).
+
+- type: textarea
+  attributes:
+    label: Question
+    description: >-
+      Ask a question.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Motivation
```

Comment:
Do we need motivation to answer the question? I think not.

We don't need a hardware details too (and reported may feel scared that it is needed to share his hardware configuration to ask question).

If it is not a question we will ask to fill a bug/ feature.

Reply:
tmpletate from github contained M, so I left it. deleted.

as for Environment, I set it as additional, it is not mandatory, if the user needs it, he can add information in a separate block that will be more readable